### DISCUSSION TITLE
Solicitação de repositório no GitHub da APyB

### DISCUSSION THREAD

**Discussion by cecivieira** [apyb/comunidade#20] [Geral]
Boa tarde!!!

Esse ano a Org. da Python Brasil, PyLadies e AfroPython unirá esforços no desenvolvimento de um programa de bolsa viagem para a Python Brasil 2022. Seria possível a APyB disponibilizar um repositório para usarmos na gestão e documentação desse programa?

As pessoas que deveriam ter perfil de administração do repositório são: @cecivieira @lipemorais @giuliocc @julianyraiol

**Comment by rougeth** [comment]
Legal! Posso criar na organização [@pythonbrasil](https://github.com/pythonbrasil). Qual nome vocês querem para o repositório? Vou adicionar o prefixo `pybr2022-`, que já estamos usando para os repositórios do time e do site da edição de 2022.

**Comment by cecivieira** [comment]
Massa. O título é: Programa de Bolsa de Viagem para a Python Brasil 2022

**Reply by rougeth** [reply]
Repositório: [@pythonbrasil/pybr2022-bolsa-viagem](https://github.com/pythonbrasil/pybr2022-bolsa-viagem)
Time com perfil de administração: [@pybr2022/time-bolsa-viagem](https://github.com/orgs/pythonbrasil/teams/time-bolsa-viagem/members)

**Comment by cecivieira** [comment]
Tu poderia mudar a branch padrão para a main, por favor?

**Comment by cecivieira** [comment]
E configura o template da issue, por favor

**Reply by rougeth** [reply]
O diretório para os templates de issues é `.github/ISSUE_TEMPLATE`, acho que se renomear a pasta que já está lá, usando letras maiúsculas, resolve o problema.

Branch padrão já é o `main`, não precisei fazer nada.

**Reply by cecivieira** [reply]
Massa!

Acho que quando criei a branch main, o github entendeu a substituição.
Renomeei a pasta e funcionou. Obrigada!
